service_files/SyncBMCData_stunnel.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Stunnel for Redundant BMC Data Synchronization
+Wants=xyz.openbmc_project.State.BMC.Redundancy.Sibling.service
+After=xyz.openbmc_project.State.BMC.Redundancy.Sibling.service
+Wants=xyz.openbmc_project.Control.SyncBMCData.service
+Before=xyz.openbmc_project.Control.SyncBMCData.service
+Wants=SyncBMCData_rsync.service
+After=SyncBMCData_rsync.service
+
+[Service]
+ExecStartPre=/usr/libexec/phosphor-data-sync/update_sibling_bmc_ip.sh
```

Comment:
What if the sibling BMC isn't available at this point, but shows up later after this BMC has already became active?  (Static IPs would make this easier.)